cloudflare_ddns/cloudflare.py

```diff
@@ -144,8 +148,8 @@ def create_record(self, dns_type, name, content, **kwargs):
         else:
             data['proxied'] = True
         content = self.request(
-            urllib.parse.urljoin(self.api_url, self.zone['id'], self.zone['id'] + '/dns_records'),
-            'put',
+            self.api_url + self.zone['id'] + '/dns_records',
+            'post',
```

Comment:
Sorry, please ignore. Looked the wrong line thinking it was in update_record method.